executor.sh

```diff
@@ -214,9 +214,9 @@ for osVer in "${os[@]}"
          do
             build_image "$osVer" "$lang" "$langVer"
             test_image
-            # if [ "$should_push" = true ];then
-            #   push_image "$osVer" "$lang" "$langVer"
-            # fi
+            if [ "$should_push" = true ];then
```

Comment:
1. is it a comparison or assignment operator here. as far as i understand it should be `"$should_push" == true`
2. add a space between semicolon and `then`